docs/cimpl6ToolingReference.md

```diff
@@ -0,0 +1,692 @@
+# CIMPL 6.0 Tooling Reference
+
+>**Note:** This documentation is a draft.
+
+_This is a comprehensive guide to CIMPL 6.0 Tooling, including the command line interface, auxiliary files, and configurations needed to produce a FHIR Implementation Guide (IG). If you're looking for a quick introduction to CIMPL and `SHR-CLI` environment setup, try the [Hello World](cimpl6Tutorial_helloWorld.md) or [In-Depth Tutorials](cimpl6Tutorial_detail.md). For details of the CIMPL language itself, see the [CIMPL Language Reference Manual](cimpl6LanguageReference.md)._
+
+***
+
+**Table of Contents**
+[TOC]
+
+***
+
+## Overview
+
+This reference manual describes the configurations, files, and commands needed to create a FHIR IG from CIMPL (**C**linical **I**nformation **M**odeling **P**rofiling **L**anguage). It assumes that the CIMPL Language files (classes, value sets, and maps) have been defined (see [CIMPL Language Reference Manual](cimpl6LanguageReference.md) for details). It also assumes that the CIMPL SHR-CLI tooling has been installed according to the directions in the [Setup and Installation Guide](cimplInstall.md).
+
+The CIMPL Tooling, also called SHR-CLI (Standard Health Record Command Line Interface), is the engine that imports a set of inputs, including CIMPL language files, and exports FHIR and other outputs, as shown below:
+
+#### _CIMPL Tooling Overview_
+
+![CIMPL Tooling Overview](img_cimpl/cli-overview.png)
+
+### Inputs
+
+* [CIMPL Language files](cimpl6LanguageReference.md), including class files, value set files, and mapping files that define your clinical information model,
+* A [Configuration file](#configuration-file) that contains directives to the tooling, and points to other resources,
+* An optional [Content Profile](#content-profile-file) file, which specifies Must-Support elements and profiling options specific to an IG,
+* One or more [Front Matter](#front-matter-files) files, which are the narratives and graphics that introduce the IG,
+* [FHIR Examples](#fhir-examples) that are to be included in the IG, and
+* A [Package List](#package-list) that has information required for building the IG.
```

Comment:
Link broken. I think it's (#package-list-file) instead of (#package-list).

Reply:
Found 2 instances. Corrected.